src/app/loadout/reducer.ts

```diff
@@ -11,52 +11,22 @@ import {
   LoadoutItem as DimApiLoadoutItem,
   DestinyVersion,
 } from '@destinyitemmanager/dim-api-types';
-import { currentProfileSelector, apiPermissionGrantedSelector } from 'app/dim-api/selectors';
+import { currentProfileSelector } from 'app/dim-api/selectors';
 import { emptyArray } from 'app/utils/empty';
 
-const reportOldLoadout = _.once(() => ga('send', 'event', 'Loadouts', 'No Membership ID'));
-
 /** All loadouts relevant to the current account */
 export const loadoutsSelector = createSelector(
   currentAccountSelector,
   currentProfileSelector,
-  (state: RootState) => state.loadouts.loadouts,
-  apiPermissionGrantedSelector,
-  (currentAccount, profile, legacyLoadouts, apiPermissionGranted) =>
-    apiPermissionGranted
-      ? profile
-        ? Object.values(profile?.loadouts).map((loadout) =>
-            convertDimApiLoadoutToLoadout(
-              currentAccount!.membershipId,
-              currentAccount!.destinyVersion,
-              loadout
-            )
+  (currentAccount, profile) =>
+    profile
+      ? Object.values(profile?.loadouts).map((loadout) =>
```

Comment:
object values errors on `undefined`, so this will error if profile is undefinedish.
the ? doesn't actually suppress any potential errors unless a fallback value is provided

Reply:
Good catch. In this case the `profile?` is unnecessary because we're already conditional on it being truthy.